lib/compare.js

```diff
@@ -24,6 +24,11 @@ const equalObject = (obj1, obj2) => {
   return true;
 };
 
+const equalFunction = (func1, func2) => {
+  if (func1 === null || func2 === null) return func1 === func2;
```

Comment:
Maybe `if (func1 === null && func2 === null) return true;`

Reply:
It's the same as in other such functions (https://github.com/metarhia/metatests/pull/95/files#diff-4188dbce06d21f3a05e4e5edb8aa4889R14).
This change would require additional checks after it to avoid null errors.

Reply:
Can't understand

Reply:
If I change this line to `if (func1 === null && func2 === null) return true;`
then the next one `return func1.toString() === func2.toString();` will potentially fail with 'calling toString on a null' if func1 is null and func2 is not and vice versa.

Reply:
Agree